### ISSUE TITLE
Allow focus and selection of interactive elements via keyboard

### ISSUE THREAD

**Issue by blissdev** [fullcalendar/fullcalendar#6465]
### Checklist

Please mark these items with an [x]

  - [x] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [x] Other people will find this feature useful

### Feature Description

Add functionality that would either be on by default, or enabled through configuration, such that users would be able to tab through all interactive elements and also allow adding event handlers to keyboard interaction on those elements.

The two initial targets for allowing this interaction would be:

- All events
- All clickable links (dates, etc)

To do this, the aforementioned elements should be added to the tab ring. Then hooks should be enabled to allow binding handlers to keyboard interaction (as well as clicking).

I see this is a specific, actionable component of https://github.com/fullcalendar/fullcalendar/issues/2536. Additionally it maybe duplicative of https://github.com/fullcalendar/fullcalendar/issues/3364 so I'd be open to switching to that if preferred.

**Comment by blissdev** [comment]
Additionally I would be willing to work on this. I've begun to look into it, but any feedback and guidance would be much appreciated.

**Comment by arshaw** [comment]
Thanks for this ticket! I've created a new one with more implementation details. We will use it from now on:
https://github.com/fullcalendar/fullcalendar/issues/3364
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?
